initfiles/etc/DIR_NAME/environment.xml.in

```diff
@@ -17,12 +17,13 @@
  <Software>
   <DropZone build="_"
             buildSet="DropZone"
-            computer="localhost"
             description="DropZone process"
             directory="${RUNTIME_PATH}/mydropzone"
             name="mydropzone"
             ECLWatchVisible="true"
-            umask="022"/>
+            umask="022">
+            <ServerList name="ServerList" server="localhost"/>
```

Comment:
Is "localhost" really can be used as a server address? How this can be resolve in multinode system?